app/helpers/application_helper.rb

```diff
@@ -53,4 +53,49 @@ def render_index_grid(grid)
     no_records = grid.current_page_records.length < 1
     render "admin/common/grid", no_records: no_records, grid: grid
   end
+
+  # Returns the new disbursment url, as well as addtional loan and org information.
+  # This information maybe be required, if the loan does not exist in the old system.
+  def labase_new_disbursment_url(loan:)
```

Comment:
The url that gets generated is pretty long and verbose, I opted to not include an example in the URL. They look like this: `http://internal.labase.org/LoanSchedule.php?LoanID=1460&loan%5Bamount%5D=160000.0&loan%5Bcurrency_id%5D=1&loan%5Bdivision_id%5D=13&loan%5Bid%5D=1460&loan%5Blength_months%5D=6&loan%5Bname%5D=Project%20with%20Gr%C3%A1fica%2022%20de%20Mayo&loan%5Borganization_id%5D=454&loan%5Borganization_snapshot_id%5D=1325&loan%5Bprimary_agent_id%5D=255&loan%5Bpublic_level_value%5D=hidden&loan%5Brate%5D=32.5&loan%5Bsecondary_agent_id%5D=145&loan%5Bsigning_date%5D=2016-10-03&loan%5Bstatus_value%5D=prospective&organization%5Balias%5D=La%20Cabril&organization%5Bcity%5D=Avellaneda&organization%5Bcountry_id%5D=1&organization%5Bdivision_id%5D=2&organization%5Bemail%5D=quintanacarlos68%40gmail.com&organization%5Bid%5D=454&organization%5Bindustry%5D=Gr%C3%A1fica&organization%5Blegal_name%5D=Cooperativa%20de%20Trabajo%20Gr%C3%A1fica%2022%20de%20MayoLimitada&organization%5Bname%5D=Gr%C3%A1fica%2022%20de%20Mayo&organization%5Bneighborhood%5D=&organization%5Bprimary_phone%5D=4208%201150&organization%5Breferral_source%5D=Red%20Gr%C3%A1fica&organization%5Bsector%5D=Gr%C3%A1fica&organization%5Bstate%5D=&organization%5Bstreet_address%5D=Rivadavia%20700%20%28Avellaneda%29&organization%5Btax_no%5D=30714015458`

And like this when decoded by chrome:
![internal_labase_org_loanschedule_php_loanid_1460_loan_5bamount_5d_160000_0_loan_5bcurrency_id_5d_1_loan_5bdivision_id_5d_13_loan_5bid_5d_1460_loan_5blength_months_5d_6_loan_5bname_5d_project_with_grafica_22_de_mayo_loan_5borganization_id_5d](https://cloud.githubusercontent.com/assets/59398/19915541/b43ea156-a078-11e6-8992-a73161f11f46.png)